### ISSUE TITLE
Bump version of `syn`

### ISSUE THREAD

**Issue by CGMossa** [jakobhellermann/bevycheck#3]
Hey! Lovely crate, but in order to make it work:

I had to bump `syn` to `syn = "1.0.72"`. Then I had to go to `bevy` and update the `syn` used there to `syn = "1.0.72"`, and finally the crate would compile with other things I had as well.
The problem is that `standard-dist` also uses a newer `syn` crate.

**Comment by jakobhellermann** [comment]
That's weird, I depend on `syn = `1.0` which if I understand correctly means every dependency semver-compatible to `1.0`, so every version `1.x.y`. What is the error message you get?

**Comment by CGMossa** [comment]
Right, I'm working off main branch of bevy. So this is a future thing.

**Comment by jakobhellermann** [comment]
While `bevycheck` depends on `bevy = 0.5`, I think it should work with bevy from the `main` branch.
Try adding
```toml
[patch.crates-io]
bevy = { git = "https://github.com/bevyengine/bevy", branch = "main" }
```
to your `Cargo.toml`, this will override the bevy dependency to use bevy main aswell.
